packages/paste-core/components/list/src/display/OrderedDisplayListContext.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from "react";
+
+import type { OrderedDisplayListVariants } from "../types";
+
+interface OrderedDisplayListState {
+  variant: OrderedDisplayListVariants;
+}
+
+export const OrderedDisplayListContext = React.createContext<OrderedDisplayListState>({ variant: "heading20" });
```

Comment:
should this be updated to `heading30` default?

Reply:
Good catch